### DISCUSSION TITLE
AWS quotas when going multi-region after Reference Architecture deployed

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#429] [Help] [answered]
Once my RefArch is deployed, and I would like to duplicate that infrastructure in new regions, are there AWS quotas I will run into?

---

<ins datetime="2022-05-23T19:58:51Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108652">Tracked in ticket #108652</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
If you are using the CIS Reference Architecture, then you may run into quota issues around NACL rules per VPC when deploying the VPC. If you do, you will want to increase that to at least 40.
